src/libopenrave/robotmanipulator.cpp

```diff
@@ -854,12 +854,7 @@ bool RobotBase::Manipulator::IsChildLink(const KinBody::Link &link) const
             return false;
         }
     }
-    for(size_t ijoint = 0; ijoint < probot->GetJoints().size(); ++ijoint) {
-        if( probot->DoesAffect(ijoint,ilink) && !probot->DoesAffect(ijoint,iattlink) ) {
-            return true;
-        }
-    }
-    return false;
+    return true;
```

Comment:
if `link` and `ee` are connected with static joint, `__pEffector->IsRigidlyAttached(link)` returns true, so this change doesn't matter. the change here is for this function to have the same behavior as other functions for `links that are connected with passive but non-static joints`

Reply:
Right, passive joints (not static).

If the behavior change is intended one, I think the function name ``IsChildLink`` is a bit strange, because ``link`` here is more like a sibling than a child of ``ee``. I'll leave to @rdiankov about the function name.